Assets/Heart/Modules/Pool/SharedAssetReferencePoolAsync.cs

```diff
@@ -83,16 +83,13 @@ public static async UniTask PrewarmAsync(this AssetReferenceGameObject original,
         public static void Dispose(this AssetReferenceGameObject original)
         {
             if (original == null) throw new ArgumentNullException(nameof(original));
-            var pool = GetOrCreatePool(original);
+            if (!Pools.TryGetValue(original, out var pool)) throw new KeyNotFoundException();
```

Comment:
The purpose of this PR is InputSystem so there should be no modifications to unrelated parts.

Reply:
Removed Pool Changes

Reply:
it still here, 
![image](https://github.com/user-attachments/assets/e216cecc-32fe-4e0a-b9db-ee4fbc3be25b)

maybe you need rollback commit and remove changed for pool

Reply:
It still here

![image](https://github.com/user-attachments/assets/2cde49e3-4381-49a1-acb0-d40f4638a417)

You need rollback commit and remove changed for pool